app/src/main/java/com/github/studydistractor/sdp/ui/HistoryScreen.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.studydistractor.sdp.ui
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.*
+import com.github.studydistractor.sdp.history.HistoryInterface
+import com.github.studydistractor.sdp.ui.components.MessageCard
+
+/**
+ * Author: Bluedrack

```

Comment:
You can rewrite the comment from the activity :
Screen showing the historic of the current logged user fetching data online using the HistoryInterface.

Reply:
Yes sorry I missed that